root/static/scripts/common/utility/formatDatePeriod.js

```diff
@@ -12,32 +12,32 @@ import ko from 'knockout';
 import formatDate from './formatDate';
 
 function formatDatePeriod<+T: {...DatePeriodRoleT, ...}>(entity: T) {
-  let {begin_date, end_date, ended} = entity;
+  let {beginDate, endDate, ended} = entity;
```

Comment:
And also here the [destructuring assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) requires you to give precise property name which means `beginDate` in this case is actually `undefined`.

Reply:
Yep, these can't be changed like this either. You'll need to define `beginDate` and `endDate` as aliases in the destructuring assignment here... see the section "Assigning to new variable names" in the article @ABCbum linked. :)